components/atoms/space/__tests__/stories.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Space
+ */
+
+// React
+import React from 'react'
```

Comment:
@mhsnprvr3 As Kristian pointed out we do not need React anymore, lets not add in, and refactor away in other places